homeassistant/components/smhi/weather.py

```diff
@@ -158,8 +160,7 @@ async def async_update(self) -> None:
         if self._forecasts:
             self._attr_temperature = self._forecasts[0].temperature
             self._attr_humidity = self._forecasts[0].humidity
-            # Convert from m/s to km/h
-            self._attr_wind_speed = round(self._forecasts[0].wind_speed * 18 / 5)
+            self._attr_wind_speed = round(self._forecasts[0].wind_speed)
```

Comment:
Is the round really needed? I dont know backend handles this.

Reply:
Also, does this make it a breaking change?

Reply:
> Is the round really needed? I dont know backend handles this.

No, It says float so I have removed it so it comes with one decimal which also frontend will show

> Also, does this make it a breaking change?

I thought no initially but then I think people are making templates and things off this so it would be a breaking change. I added the section so please add the label for it.